packages/shared/src/hooks/analytics/useAnalyticsSharedProps.ts

```diff
@@ -19,10 +19,11 @@ export default function useAnalyticsSharedProps(
   // Use ref instead of state to reduce renders
   const sharedPropsRef = useRef<Partial<AnalyticsEvent>>();
   const { query } = useRouter();
-  const { flags } = useContext(FeaturesContext);
   const { themeMode, spaciness, insaneMode } = useContext(SettingsContext);
   const { visit, anonymous, tokenRefreshed, user } = useContext(AuthContext);
   const [sharedPropsSet, setSharedPropsSet] = useState(false);
+  const { growthbook } = useGrowthBookContext();
+  const flags = growthbook.getFeatures();
```

Comment:
We actually decided to remove these, as it will enroll people by default.
But maybe @idoshamun this way could potentially work? (not sure how you feel about it)

Reply:
Yap you're right